packages/runtime/README.md

```diff
@@ -2,6 +2,24 @@
 
 The botpress runtime is a lightweight version of botpress that runs on the cloud
 
+## Testing Runtime with Messaging
+
+1. Edit the file `dist/.env` and configure the following elements:
+
+```js
+ENABLE_EXPERIMENTAL_CONVERSE=true
+MESSAGING_ENDPOINT=http://localhost:3100
+ENABLE_EXPERIMENTAL_SOCKET=true
+```
```

Comment:
`ENABLE_EXPERIMENTAL_SOCKET` is not required. In fact I don't think this config exists anymore anywhere. The `ENABLE_EXPERIMENTAL_CONVERSE` options should just be removed from the runtime and be enabled by default (I'll do this in a future PR when https://github.com/botpress/botpress/pull/11646 is merged)

Reply:
@allardy the tool we have to test the webchat on messaging can also work with the runtime. What you are doing here is basically re-creating the same exact test environment but instead of being on the messaging repo it's here. 

In any case, I don't mind having this in the runtime, but I think it could be better in the long term if we were to only focus on one documented and maintained way of  testing the webchat/

Reply:
@laurentlp The tool on the messaging repo tests messaging. This tool's purpose is only to test the runtime, but since messaging is necessary to test the runtime, it also sets up everything so testing can go smoothly.

Also, since messaging runs a different node version than the runtime, I don't want to keep changing node versions when starting botpress or starting the tool to test the messaging/runtime combination (nvmrc doesn't work on windows, switching version is a pain)